packages/venia-concept/src/shared/durations.js

```diff
@@ -0,0 +1,5 @@
+export const drawer = 192;
```

Comment:
Good idea. This is actually the duration of the drawer's closing animation, though. The opening animation has a different duration, so maybe this could be `drawerClose` or something.